src/components/coffee-input.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import {addCoffee} from '../database/coffee';
+import {FancyInput, FancyDropdown, FancyButton} from './input-stuff.js'
+
+
+// I'm using the same name...that feels wrong. Is it going to break it?
+// or just bad behavior?
+const FancyForm = (props) => {
```

Comment:
reusing this name breaks nothing! it's great, that's one of the benefits of modules, you can have whatever you want in this file and it won't collide with anything outside it